recipes/bitserializer/all/conanfile.py

```diff
@@ -102,7 +102,7 @@ def validate(self):
         compiler_name = str(self.settings.compiler)
         if compiler_name == "gcc" and self.settings.compiler.libcxx != "libstdc++11":
             raise ConanInvalidConfiguration(f'Using {self.ref} with GCC requires "compiler.libcxx=libstdc++11"')
-        elif compiler_name == "clang" and self.settings.compiler.libcxx not in ["libstdc++11", "libc++"]:
+        elif compiler_name == "clang" and self.settings.compiler.libcxx not in ["libstdc++11", "libc++", "c++_static"]:
             raise ConanInvalidConfiguration(f'Using {self.ref} with Clang requires either "compiler.libcxx=libstdc++11"'
                                             ' or "compiler.libcxx=libc++"')
```

Comment:
```suggestion
```

I would suggest just lifting this restriction, as using `libstdc++` is only worthwhile for legacy compiler versions like GCC <=5